.gitignore

```diff
@@ -215,6 +217,7 @@ build/bin/sage-build-env-config
 /pkgs/sagemath-categories/requirements.txt
 /pkgs/sagemath-environment/requirements.txt
 /pkgs/sagemath-repl/requirements.txt
+/pkgs/sagemath-brial/requirements*.txt
 /pkgs/sagemath-categories/MANIFEST.in
 
```

Comment:
OK for now. But I think this block of lines should be sorted out by distribution names...